src/Core.tsx

```diff
@@ -280,6 +283,11 @@ export class Core extends React.Component<IModalProps, IModalState> {
     // send disconnect method to wallet connect
     this.disconnectWC(provider)
 
+    // portis
+    if (provider.isPortis) {
+      provider._portis.logout()
```

Comment:
We should await this, it is a `Promise` - https://github.com/portis-project/web-sdk/blob/develop/packages/portis-web3/src/index.ts#L126

should we move this to a `disconnectPortis` method? As for WC

Reply:
I see now WC `disconnect` returns a `Promise` too
https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/packages/ethereum-provider/src/index.ts#L116

Reply:
Great! I have added a `disconnect` method to the wrapper. This mirrors how walletconnect works. So, now when disconnecting, I can check if this method exists and call it.